web/themes/joinup/prototype/sass/components/_listing.sass

```diff
@@ -14,9 +14,12 @@
 .listing__item
   position: relative
   background: #fff
+  display: flex
+  align-items: stretch
 
 .listing__card
-  position: static
+  display: flex
```

Comment:
I think the title to content positioning should be different comparing with the wireframes:

Current:
![video-featured](https://user-images.githubusercontent.com/1923476/29451881-8edfb0c4-8404-11e7-854e-83ed3e9204e5.png)

Wireframe:
![video-tile](https://user-images.githubusercontent.com/1923476/29451885-924a0c64-8404-11e7-8a79-60fb31e02e2e.jpg)

The title is aligned to top, content is aligned in the middle.